model/task_queue.go

```diff
@@ -574,10 +574,13 @@ func taskQueueGenerationRuntimePipeline() []bson.M {
 				"_id": 0,
 				"distroQueue": bson.M{"$push": bson.M{
 					"k": "$" + taskQueueDistroKey,
-					"v": bson.M{"$subtract": []interface{}{
-						"$" + bsonutil.GetDottedKeyName(taskQueueDistroQueueInfoKey, taskQueueInfoPlanCreatedAtKey),
-						"$" + taskQueueGeneratedAtKey}},
-				}}},
+					"v": bson.M{"$multiply": []interface{}{
+						1000000,
```

Comment:
Magic number?

Reply:
To add on to this, it would be helpful to add a comment here explaining what units you're converting from/to with this operation